### DISCUSSION TITLE
Update FAQ

### DISCUSSION THREAD

**Discussion by IceflowRE** [microsoft/vcpkg#20708] [Q&A]
As noted in this [message](https://github.com/microsoft/vcpkg/issues/11684#issuecomment-791791485) the FAQ should be updated as they are a bit outdated.
This issue can can be used to collect all issues with it.

From my side especially [why not conan](https://vcpkg.readthedocs.io/en/latest/about/faq/#why-not-conan) is horrible outdated.
- Public federation vs private federation

---

Related issues:
- #11684

**Comment by PhoebeHui** [comment]
@IceflowREm, sorry I forget to reopen the old issue, and thanks for reporting this issue again!

**Comment by strega-nil-ms** [comment]
Yeah, this is incredibly outdated; in my opinion the answer is to just nuke it. It is extremely outdated and misleading on both sides.
